### PR TITLE
Eliminate BuildSupport Repo

### PR DESCRIPTION
**Pull request by Myoldmopar**
This is fairly gross, but eliminates the need for the separate, always out-of-sync-at-the-wrong-time buildsupport repo.  This adds a bunch of binary blobs to this repo, but they are almost wholly static binary blobs that would go away should we decide to change them to a compilable source.  This PR eliminates the install_remote CMake command as all package resources are either included here in the bin/ folder, or compiled from source.

ToDo:
- [x] Test package builds on all 3 platforms
- [x] Evaluate if we need a plist id generator function similar to install_remote_plist
- [x] Test package builds on CI (Mac will probably still fail for other reasons...)

### CODE REVIEW

**Comment by Myoldmopar** [comment]
@mjwitte @JasonGlazer FYI, tools like IDF editor now have the original source in `src/IDF_Editor`, while the binary blobs are in `bin/IDFEditor`.  No more commits to BuildSupport.

**Comment by JasonGlazer** [comment]
@Myoldmopar So is everything that was in buildsupport here now?

**Comment by Myoldmopar** [comment]
Yep, we can now make standalone packages without having to maintain BuildSupport.  A bit gross because of having to bring binary blobs over, but frankly not any _more_ gross than having to manually update BuildSupport and try to keep them in sync without tracking the state of BuildSupport.  A net gain.

**Comment by mjwitte** [comment]
@Myoldmopar Are we going to keep BuildSupport as an archive so we can reference the change history?  I would guess ultimately you want to delete that repo someday.  But we don't want to loose the PR history, do we?

**Comment by Myoldmopar** [comment]
My plan would be to make a single commit that deletes everything except a README, which would describe the history/purpose of the repo.  The entire version history would be there, and the issues/pull requests would still have a history.  But it would appear as an empty repo.

**Comment by nrel-bot-3** [comment]
@Myoldmopar @lgentile it has been 14 days since this pull request was last updated.

**Comment by nrel-bot-3** [comment]
@Myoldmopar @lgentile it has been 14 days since this pull request was last updated.

**Comment by nrel-bot-3** [comment]
@Myoldmopar @lgentile it has been 15 days since this pull request was last updated.

**Comment by Myoldmopar** [comment]
I've tested this out building numerous packages and things are awesome.  In the process we even got Mac building packages for us beautifully.  I don't think we need any plist manipulation since we aren't doing install_remote anymore.  The installation still seems to put EP-Launch_Lite in the wrong folder if you have a previous installation, but that was happening before.  I am going to merge this, and BuildSupport is officially not-needed.

**Comment by JasonGlazer** [comment]
@Myoldmopar I think the wrong version of EP-Launch ended up being moved into the EnergyPlus repo. The current develop is has source that shows EP-Launch is at version 2.10

https://github.com/NREL/EnergyPlus/blob/develop/src/EP-Launch/frmAbout.frm#L78

But I found the EP-Launch version 2.11 which was released as part of EnergyPlus 8.5.0 under the "8.5.0" tag under the EnergyPlusBuildSupport repo

https://github.com/NREL/EnergyPlusBuildSupport/blob/8.5.0/src/EP-Launch/frmAbout.frm#L78

Did the "8.5.0" ever get merged into master in BuildSupport repo?

**Comment by Myoldmopar** [comment]
Well, the master branch of BuildSupport was always a flaky situation, so I tried to be careful, but I wouldn't be shocked if something was mistaken.  I'd say just push your latest version of EP-Launch into here and not sweat what was in BuildSupport.

**Comment by JasonGlazer** [comment]
@Myoldmopar @mjwitte I updated EP-Launch to the version that went out with 8.5.0 as the first commit on #5840. 

I also noticed that RunDirMulti.bat was not the latest. It was updated at BuildSupport  [#28](https://github.com/NREL/EnergyPlusBuildSupport/pull/28)

Given how flaky the BuildSupport directory was, I am going to check the rest of the utilities and make sure they are at the latest versions. I will make it a new pull request.

**Comment by JasonGlazer** [comment]
@Myoldmopar and @mjwitte I looked at other utilities that traditionally I have been involved with and here is my assessment:
- AppGPostProcess (source in fortran) – source ok, BINARY MISSING I did not see a project that compiles it in the Visual Studio solution
- EP-Launch (source) needed updating – fixed yesterday as part of EP-launch update
- IDFVersionUpdater (source in XOJO) – source ok, binary same size
- IDF Editor (source in VB) – SOURCE NEEDS UPDATING, BINARY NEEDS UPDATING
- Epmacro (binary) – binary ok. ERROR: “/bin/EPMacro/Windows contains EP-Compare directory”
- RunDirMulti.bat (source) needed updating – fixed yesterday as part of EP-Launch update
- RunEplus.bat (source batch) - ok
- EP-Compare (source in XOJO) – SOURCE MISSING (it was before also) – windows binary ok
- EPDraw (source in XOJO) – SOURCE MAY NEED UPDATING have 0.13 on my computer but 0.10 was what have been getting distributed for years.
- ParametricPreprocessor (source in Fortran) – Source in repo newer than what I have
- ExpandObjects (source in Fortran) – source in repo newer than what I have
- HVAC-Diagram (source in Fortran) – source in repo newer than what I have

In summary:
- AppGPostProcess needs a binary but I am not sure how you want to produce it
- IDF Editor needs updating (which I will do soon)
- The /bin/EPMacro/Windows directory has a copy of some of the EP-Compare binaries, probably a copyo (like a typo?) - do you want me to clean it up?
